### DISCUSSION TITLE
Xcode 14 beta with macos 13 sdk not available?

### DISCUSSION THREAD

**Discussion by greeshmay** [actions/virtual-environments#6211] [Installed software]
Until yesterday, we were able to use Xcode 14 beta to build our project that depended on macos 13 sdks. But with the most recent release, we are no longer able to build our project. I see that there is a PR to include Xcode 14 beta with the right sdks - https://github.com/actions/runner-images/pull/6204. Do you know when this will be released? All our builds have been failing for over 24 hours.

**Comment by al-cheb** [comment]
https://github.com/actions/runner-images/issues/6199#issuecomment-1240969957

**Comment by hakonk** [comment]
Will there also be a private beta program like before to get access to MacOS 13 Ventura?

**Reply by miketimofeev** [reply]
@hakonk we hope to open macOS-13 as a public beta in the following months, please keep an eye on the announcements.
